discover/portal/articles-dxp/140-workflow/02-kaleo-forms/02-assigning-kaleo-forms.markdown

```diff
@@ -0,0 +1,160 @@
+
+Next create a form named *Spa Order Process--Technician Claim Form* for the
+Technician Claim workflow task. Follow the same procedure used above, this time
+deleting these fields from the full field set:
+
+- Approved
+- Managerial Comments
+
++$$$
+
+**Example Use Case Note:** In the Spa Order Process, the Managerial Approval
+workflow task and its associated form are filled out by the spa manager. The
+manager can oversee the order, making sure that customers requests are being met
+and that all technicians are fulfilling their share of the spa's duties.
+
+$$$
+
+Create a form to be used in the Managerial Approval task, called *Spa Order
+Process--Managerial Approval Form*. The manager can change anything she wants
+about the order, so include the entire field set in the form.
+
++$$$
+
+**Example Use Case note:** The customer service representative finalizes the
+order and gets back to the customer on the final details (available time, name
+of technician, and confirms the services they want).
+
+$$$
+
+There's one more form to create: the *Spa Order Process--Final Order Form*. Like
+the last form, it should have all the form's fields so that the customer service
+representative can review the order with the customer.
+
+![Figure 7: Once created, your process is listed in Kaleo Forms.](../../../images-dxp/kaleo-forms-processes.png)
+
+Click *Save* when finished. The process is ready to test.
+
+
+## Test the Process [](id=test-the-process)
+
+Once your new process is developed, add records to it by clicking on the process
+name and then the Plus button (![Add](../../../images-dxp/icon-add.png)).
+
+The application displays the form you assigned to the workflow's initial state.
+Fill out the form and click Save.
+
+![Figure 8: Fill out the form and send it through the process.](../../../images-dxp/kaleo-forms-new-process-form.png)
+
+Once the initial form is submitted in the Kaleo Forms application, the process
+is managed by the workflow engine. In a process backed by the Spa Order
+Workflow definition, the workflow moves first to the Technician Claim task, then
+to the Managerial Approval task, and last, to the Final Order task. Whatever
+users or roles you assigned to the tasks receive notifications, and the task
+appears in the Assigned to My Roles section of the My Workflow Tasks portlet. A
+notification is also triggered in the Notifications application. Once in the
+task, the user views and approve the form or clicks the edit button. At this 
+point the workflow task forms you created come into play. Each assigned user fills out the form, saves it, and sends it along in the workflow.
+
+The Kaleo Forms application unites two powerful features of Liferay by
+integrating a workflow and a form into a new entity, the Kaleo Process.
+
+## Related Topics [](id=related-topics)
+
+[Kaleo Designer](/discover/portal/-/knowledge_base/7-0/kaleo-designer)
+
+[Using Workflow](/discover/portal/-/knowledge_base/7-0/enabling-workflow)
```

Comment:
_Using Workflow_ is the name of the article for 7.0, but this article was renamed to _Activating Workflow_ for 7.1. 😉